pytify/cli.py

```diff
@@ -75,6 +77,8 @@ def interaction(self):
                 if search is not False:
                     self.menu(list=self.pytify.list())
 
+def autocomplete:
```

Comment:
Shouldn't the function have paranthesis?
def autocomplete():

And is there any reason we need this function? :)